PublicMapping/districtbuilder
# Style map editor

## Overview
Styles the map editor and its associated tooltips.

### Demo
<img width="1680" alt="Screen Shot 2020-08-18 at 4 50 33 PM" src="https://user-images.githubusercontent.com/5672295/90535627-0899b080-e173-11ea-80e6-8b9a18522e57.png">
<img width="324" alt="Screen Shot 2020-08-18 at 4 51 32 PM" src="https://user-images.githubusercontent.com/5672295/90535761-2404bb80-e173-11ea-8c59-7541bd92f220.png">
<img width="466" alt="Screen Shot 2020-08-18 at 4 51 47 PM" src="https://user-images.githubusercontent.com/5672295/90535766-26671580-e173-11ea-8969-e4232fdd8fb8.png">
<img width="535" alt="Screen Shot 2020-08-18 at 4 52 08 PM" src="https:user-images.githubusercontent.com/5672295/90535787-2cf58d00-e173-11ea-9c33-698294dbcb0e.png">


### Notes
- @pcaisse : @maurizi and I tried to figure out a typescript error that I was getting on line `236` in `ProjectSidebar.tsx` but we were unsuccessful! Could you take a look? It doesn't like when we try to use one of `ThemeUIStyleObject`'s children `blankValue`, but if we:
    - Put it into its own style object,
    - or put the `style` style object into the scope of the function then things seem to work.
- Added the ability to change the size of Icons.
- Removed the custom body font.

## Testing Instructions
- `git pull`
- Go to a map (or create one if you have none)
- Note the new styles at the outset.
- Hover on table headers that don't display a full name (.e.g. "Pol."). You should see a browser title tag.
- Hover on the icon buttons. Each should have a browser title tag.
- On the map, hover on a block. You should see the styled map tooltip. Note that the population number is aligned with the bar charts.
- Draw a discontinuous district. 
- You should see the text in Population and Deviation change color.
- Approve the change.
- You should see a little red triangle icon. Hover to see the title tag.
- Hover on this row's Race chart. You should see the styled race tooltip.
- Hover on the lock icon to see the title tag. Lock this row. Hover to see the new title tag.
- In the top right, note that the select input has a default value of "Select...". Click to see the other options, and note that they are capitalized.

Closes #276
